build.assets/tooling/cmd/notarize-apple-binaries/args.go

```diff
@@ -0,0 +1,113 @@
+/*
+Copyright 2022 Gravitational, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"debug/macho"
+	"fmt"
+
+	"github.com/gravitational/kingpin"
+	"github.com/gravitational/trace"
+	"github.com/sirupsen/logrus"
+)
+
+const binaryArgName string = "binaries"
+
+type Args struct {
+	logLevel      *string
+	logJSON       *bool
```

Comment:
Given that this is `package main` there's really no difference between exporting fields or not exporting fields, but I'm curious why some of these are exported and others aren't.

Reply:
I simply typo'd these when defining the struct. Without access modifier keywords I am significantly more likely to make this mistake in golang vs another strongly typed language with that language feature (i.e. C#, Java).